.github/tests/selenium/webdriver/webdriver.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-VERSION=$(
-	dpkg -s google-chrome-stable | grep Version | awk '{print $2}' | sed 's/-.*//'
-)
+# VERSION=$(
+# 	dpkg -s google-chrome-stable | grep Version | awk '{print $2}' | sed 's/-.*//'
+# )
+VERSION=124.0.6367.91
```

Comment:
I'm getting https://storage.googleapis.com/chrome-for-testing-public/124.0.6367.118/linux64/chromedriver-linux64.zip but the current and latest version is: https://storage.googleapis.com/chrome-for-testing-public/124.0.6367.91/linux64/chromedriver-linux64.zip at https://googlechromelabs.github.io/chrome-for-testing/#stable

Reply:
If dpkg stable version is 118, use that, no problems. I'd better use what they think is stable rather than a fixed version string here.

Reply:
I'm getting error:
```
Connecting to storage.googleapis.com (storage.googleapis.com)|142.251.163.207|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-05-02 17:54:24 ERROR 404: Not Found.
```
with version 118 here: https://storage.googleapis.com/chrome-for-testing-public/124.0.6367.118/linux64/chromedriver-linux64.zip